bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx82.html

```diff
@@ -40,7 +40,7 @@ <h1 class="wnp-main-title">Firefox protects your browsing. <strong>Mozilla VPN</
 
         <div class="c-picto-block">
           <div class="c-picto-block-image">
-            {% include 'firefox/whatsnew/includes/illo-vpn-laptop.svg' %}
```

Comment:
Renamed these images to match the new convention for the inline dark/light mode SVGs. 
- `bedrock/firefox/templates/firefox/whatsnew/includes/fx82/illo-vpn-mobile.svg`
- `bedrock/firefox/templates/firefox/whatsnew/includes/fx82/illo-vpn-laptop.svg`